pkg/controller/toolchainstatus/toolchainstatus_controller.go

```diff
@@ -516,6 +586,20 @@ func (r *ReconcileToolchainStatus) setStatusUnreadyNotificationCreationFailed(re
 		})
 }
 
+func (r *ReconcileToolchainStatus) setStatusReadyNotificationCreationFailed(reqLogger logr.Logger,
+	toolchainStatus *toolchainv1alpha1.ToolchainStatus, message string) error {
+
+	return r.updateStatusConditions(
+		reqLogger,
+		toolchainStatus,
+		toolchainv1alpha1.Condition{
+			Type:    toolchainv1alpha1.ConditionReady,
+			Status:  corev1.ConditionTrue,
+			Reason:  "Status Ready Notification creation failed",
```

Comment:
1. The ready status should be set to false if there is a problem. Not to true.
2. Reason is a machine readable string. It's like a type. For human friendly details about the error we use messages:
```suggestion
			Status:  corev1.ConditionFalse,
			Reason:  toolchainv1alpha1.ToolchainStatusRestoredNotificationCRCreationFailedReason,
```